### PR TITLE
finished assessment

### PR DESCRIPTION
**Pull request by KevinBurgin**

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -17,38 +25,68 @@
...
-
+//create a function called coded 
+//passing in a string
+const coded = (string) => {
+//use .replace for 'a','e','i','o' to 4,3,1,0 and return
+   return string.replace(/a/g,'4').replace(/e|E/g, '3').replace(/i/g,'1').replace(/o/g,'0')
+}
+console.log(coded(secretCodeWord1))
+console.log(coded(secretCodeWord2))
+console.log(coded(secretCodeWord3))
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:53]: YES!!! You are the only one so far to figure out this problem with regex. Great job!

File: code-challenges.test.js [hunk 2]
```diff
@@ -17,38 +25,68 @@
...
+//conditional if includes is true return value
+        if(z === true){
+            return value
+        }
+    })
+//return variable
+    return m
+}
+console.log(letter(arrayOfWords1))
+console.log(letter(arrayOfWords2))
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:89]: This is amazingly clean.

File: code-challenges.test.js [hunk 3]
```diff
@@ -65,7 +111,23 @@ var hand3 = [5, 5, 5, 5, 4]
...
+//create a function called fullHouse
+//pass in array
+const fullHouse = (array) => {
+//set counter variable
+    let counter = 0
+//use map method
+    let m = array.map((value,index)  => {
+//conditional if value equals value at index 1, and want to compare if value 1 equals value 2 and so on
+    if(value === value[1]){
+        return
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:125]: return _____?

File: code-challenges.test.js [hunk 4]
```diff
@@ -65,7 +111,23 @@ var hand3 = [5, 5, 5, 5, 4]
...
+    }
+    })
+//return counter
+    return
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:129]: return _____?

File: rails-questions.md [hunk 5]
```diff
@@ -1,39 +1,59 @@
...
 6. What is the command to generate a migration file?
 
+$ rails generate migration add_item_to_schdule
 
 
 7. What is the naming convention for generating a Rails model? What is the naming convention for the table that is generated with this command?
 
+CamelCase
+snake_case
+
```
Comment (WalkerTexas-coder) [inline rails-questions.md:42]: Whats an example of a model generation command?

File: rails-questions.md [hunk 6]
```diff
@@ -1,39 +1,59 @@
...
 
 
 8. What is the schema file in a Rails application? Can you modify the schema directly? Why or why not?
 
+The schema file in rails is the document that holds the current state of the database schema.
+You cannot modify the schema the schema is only modified by migrating
```
Comment (WalkerTexas-coder) [inline rails-questions.md:48]: Great!

**Review by WalkerTexas-coder — APPROVED** [review]
Kevin great work! I think this is some of your best work so far. I left a few comments I'd like you to respond too. I am really impressed. You have great logic and your code is super clean. I'd like to see more elaboration on the interview questions next week.

File: code-challenges.test.js (hunk 1 above)
Comment (KevinBurgin) [reply code-challenges.test.js:53]: Thank you

File: code-challenges.test.js (hunk 3 above)
Comment (KevinBurgin) [reply code-challenges.test.js:125]: yea i was struggling with this one i wasnt sure what to return

File: code-challenges.test.js (hunk 4 above)
Comment (KevinBurgin) [reply code-challenges.test.js:129]: same was confused with this question and how to approach it

File: rails-questions.md (hunk 5 above)
Comment (KevinBurgin) [reply rails-questions.md:42]: sorry this should be PascalCase
and an example would be
$rails g model Table name:string

**Pull request closed without merging**